src/containers/NFT/NFT.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router'
+import { useTranslation } from 'react-i18next'
+import NoMatch from '../NoMatch'
+import NFTHeader from './NFTHeader/NFTHeader'
+import './styles.scss'
+import {
+  analytics,
+  ANALYTIC_TYPES,
+  NOT_FOUND,
+  BAD_REQUEST,
+} from '../shared/utils'
+
+const ERROR_MESSAGES: any = {
+  [NOT_FOUND]: {
+    title: 'assets.no_nfts_message',
+    hints: ['check_nft_id'],
+  },
+  [BAD_REQUEST]: {
+    title: 'invalid_xrpl_address',
+    hints: ['check_nft_id'],
+  },
+  default: {
+    title: 'generic_error',
+    hints: ['not_your_fault'],
+  },
+}
+
+const getErrorMessage = (error: any) =>
+  ERROR_MESSAGES[error] ?? ERROR_MESSAGES.default
```

Comment:
the way that you're using this, I don't think that the default error should be part of `ERROR_MESSAGES` because it is special. It should be its own constant.